docs/mkdocs.yml

```
@@ -30,3 +30,4 @@ nav:
         - ... | mozdata/**.md
     - UDFs:
         - ... | mozfun/**.md
+    - bqetl CLI: bqetl.md
```

I was thinking we should have a reference section for some of the stuff that's currently in the README. This would go well as a sub-topic under it.

```suggestion
    - Reference: 
      - bqetl CLI: bqetl.md
```